keyvault/data-plane/azure-keyvault/src/main/java/com/microsoft/azure/keyvault/KeyVaultClient.java

```diff
@@ -11,7 +11,7 @@
  * Class for Key Vault Client.
  * 
  */
-public final class KeyVaultClient extends KeyVaultClientCustomImpl implements KeyVaultClientCustom {
+public final class KeyVaultClient extends KeyVaultClientCustomImpl {
 

```

Comment:
What are the implications of this? This is public API so need to be careful.

Reply:
`public class KeyVaultClientCustomImpl implements KeyVaultClientCustom` and
`public final class KeyVaultClient extends KeyVaultClientCustomImpl`
I thought the inheritance was transitive? So since KeyVaultClient extends that CustomImpl, it also implements that interface. The KeyVaultClient itself doesn't have any special logic that overrides any of the existing methods.

Reply:
Thanks for clarifying. I am fine with this change now.